application: serial_lte_modem: Doc update board config (#8661)
nrfconnect/sdk-nrf: applications/serial_lte_modem/doc/slm_data_mode.rst

The SLM application buffers all the arbitrary data received from the UART bus before initiating the transmission.

The transmission of the buffered data to the LTE network is triggered by the time limit when the defined inactivity timer times out.

> If there is no time limit configured, the minimum required value applies. Refer to ``#XDATACTRL`` command below.

Reviewer: ```suggestion
If there is no time limit configured, the minimum required value applies.
For more information, see the `Data mode control #XDATACTRL`_  command.
```
